pandas/tests/extension/test_arrow.py

```diff
@@ -1697,7 +1697,7 @@ def test_arith_series_with_scalar(
         elif arrow_temporal_supported:
             request.node.add_marker(
                 pytest.mark.xfail(
-                    raises=TypeError,
+                    raises=TypeError if pa_dtype != "duration[ns]" else AssertionError,
```

Comment:
duration[ns] now raises an AssertionError for this set of tests rather than a TypeError as a side effect of this change.